README.md

```diff
@@ -138,6 +138,14 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 
 `meta` will only be included in your response if you are using an Adapter that supports `root`, as JsonAPI and Json adapters, the default adapter (FlattenJson) doesn't have `root`.
 
+### Links
```

Comment:
Taking into consideration this is only available to JsonApi's Adapter I think this would not be the better place to introduce this feature.
But this is an awesome opportunity to push our docs further! (we are trying to build up some new docs!)
Would be great if instead of adding this to the README we could add a whole article on the [new docs](https://github.com/rails-api/active_model_serializers/tree/master/docs) with it, maybe with some examples and the link to the jsonAPI format. Could you do this?